lib/block_view_stake_test.go

```diff
@@ -49,17 +49,10 @@ func _testStaking(t *testing.T, flushToDB bool) {
 
 	_registerOrTransferWithTestMeta(testMeta, "m0", senderPkString, m0Pub, senderPrivString, 1e3)
 	_registerOrTransferWithTestMeta(testMeta, "m1", senderPkString, m1Pub, senderPrivString, 1e3)
-	_registerOrTransferWithTestMeta(testMeta, "m2", senderPkString, m2Pub, senderPrivString, 1e3)
```

Comment:
This is just some cleanup. No sense sending $ to test users who aren't used in the test.